src/main/scala/com.snowplowanalytics.maxmind.iplookups/IpLookups.scala

```diff
@@ -18,9 +18,8 @@ import java.net.InetAddress
 import com.maxmind.db.CHMCache
 import com.maxmind.geoip2.DatabaseReader
 import com.twitter.util.SynchronizedLruMap
-import scalaz._
-
 import model._
+import cats.implicits._
```

Comment:
you can just do `cats.syntax.either._` and you can add it to the imports above, the second group is dedicated to the ones related to snowplow

Reply:
Oh I see